charts/spsh-client/templates/ingress.yaml

```diff
@@ -1,9 +1,11 @@
+{{- if .Values.ingress.enabled -}}
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: spsh-client-ingress
+  name: {{ include "spsh-client.ingressName" . }}
+  labels:
+    app: {{ template "spsh-client.name" . }}
 spec:
-  ingressClassName: nginx
```

Comment:
I think we still need to be able to set the ingress class

Reply:
I added ingressClassName

Reply:
The ingress class should be in a variable so we can set it to something different if we don't want to use the Nginx ingress class.